src/pythonic/index.md

```diff
@@ -116,6 +116,23 @@ for conveying what you're doing.
 > Also, make sure you document your exported functions. From experience, working with 
 > libraries with non-existent documentation is not fun. Please don't add to the problem!
 
+The contents of [Cargo.toml](./pythonic/primes/Cargo.toml) are below:
+```
+[package]
+name = "primes"
+version = "0.1.0"
+authors = ["Michael Bryan <michaelfbryan@gmail.com>"]
+
+[dependencies]
+primal = "*"
+libc = "*"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+```
+
+Adding `cdylib` to the `lib` section compiles the crate as a system library
+usable by other languages.
```

Comment:
Should `rlib` be removed from the lib section?  From what you mentioned 
```A rlib on the other hand is the Rust compiler's way of representing a compiled artefact, usually a crate. It's completely implementation defined and not meant to be used by anyone other than rustc,```

It also sounds like we're building a dynamic library and not a static library.